bloom-housing/bloom
# fix(backend): enforces lower casing of emails

## Issue
https://github.com/bloom-housing/bloom/issues/1912

Addresses #issue

## Description
Addresses Issue #1912  enforcing lower casing of incoming emails when:
new users are created, a look up of a user by email address happens, during user confirmation, and user log in

## Type of change

## How Can This Be Tested/Reviewed?
Create a new user with some combination of upper and lower case letters in the email address
attempt to log in as that user but only input the email address in lower case (no upper case characters) you should be able to log in as if the cases matched

if you have DB access, peep the new record and verify the email stored in the db is lower cased

## Checklist:
